Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -402,6 +402,7 @@ extension NSManagedObjectContext: MessageDatabaseSession {
         if let parentMessageId = parentMessageId,
            let parentMessageDTO = MessageDTO.load(id: parentMessageId, context: self) {
             parentMessageDTO.replies.insert(message)
+            parentMessageDTO.replyCount += 1
```

Comment:
Interesting, I wonder if this fixes the following issue: https://github.com/GetStream/stream-chat-swift/issues/1806

Worth to double-check 🙂

Reply:
If yes, can we add this fix to the changelog?

Reply:
While they may be connected, in #1806 I believe the issue is around Publishers itself.
It is a pretty common mistake/misconception where Publishers send the value prior to the change.

I will look for a post describing this scenario

Reply:
Here it is: https://stackoverflow.com/questions/58403338/is-there-an-alternative-to-combines-published-that-signals-a-value-change-afte

Reply:
Added a comment there: https://github.com/GetStream/stream-chat-swift/issues/1806#issuecomment-1064969997